build.gradle

```diff
@@ -4,11 +4,13 @@ buildscript {
     }
     repositories {
         mavenCentral()
+        jcenter()
     }
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
         classpath("org.liquibase:liquibase-gradle-plugin:${gradlePluginVersion}")
         classpath("org.postgresql:postgresql:${postgreSqlVersion}")
+        classpath("org.asciidoctor:asciidoctor-gradle-plugin:${asciidoctorPluginVersion}")
```

Comment:
Could you please do a research whether there the way to manage such dependencies's versions in BOM?